Wbooks/src/app/screens/library/components/bookList/layout.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react'
+import {
+  View,
+  FlatList,
+} from 'react-native';
+import styles from './styles.js';
+import Book from '../book/index.js'
+
+class BookList extends Component {
+  render() {
+    return (
+    <View style = {styles.container }>
+      <FlatList
+        data = {this.props.bookList}
+        renderItem={({item}) => <Book style = {styles.listItem} book = {item}/>}
```

Comment:
We don't use arrow functions inside the render, you should call it outside the render.

Reply:
done